KuringApp/KuringApp/Assets.xcassets/AppIcon.appiconset/Contents.json

```
@@ -1,6 +1,7 @@
 {
   "images" : [
     {
+      "filename" : "kuring.app.png",
```

```suggestion
      "filename" : "kuring_app.png",
```
이름이 바뀌어야 하는건 아닌지 체크 한번 부탁드립니다

---

넵 AppIcon의 경우에는 .을 허용했지만 언더바로 통일시키는 것이 좋을 것 같아 수정했습니답!